app/graphql/mutations/create_organization_mutation.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Mutations
+  # GraphQL mutation to create a new organization and
+  # add the current user as a member
+  class CreateOrganizationMutation
+    def call(_obj, args, ctx)
```

Comment:
Changing it would be fine by me, I'm probably currently more used to JS conventions than Ruby conventions :)